loleaflet/src/core/Socket.js

```diff
@@ -1139,6 +1139,8 @@ app.definitions.Socket = L.Class.extend({
 			this._map.options.doc = docUrl;
 			this._map.options.previousWopiSrc = this._map.options.wopiSrc; // After save-as op, we may connect to another server, then code will think that server has restarted. In this case, we don't want to reload the page (detect the file name is different).
 			this._map.options.wopiSrc = encodeURIComponent(docUrl);
+			window.wopiSrc = this._map.options.wopiSrc;
```

Comment:
I thought we were killing this._map.options.wopiSrc etc. - and using these via the global settings everywhere to reduce confusion.